clients/web/src/views/body/UsersView.js

```diff
@@ -60,14 +59,27 @@ var UsersView = View.extend({
             parentView: this
         }).on('g:resultClicked', this._gotoUser, this);
 
+        if (getCurrentUser() && getCurrentUser().get('admin')) {
+            var userCountpromise = UserCollection.getUsersCount()
```

Comment:
Use `const`, and name this variable with proper camelCase (as `userCountPromise`).